pyaxmlparser/core.py

```diff
@@ -252,3 +252,17 @@ def get_effective_target_sdk_version(self):
             return int(target_sdk_version)
         except (ValueError, TypeError):
             return 1
+
+    @property
+    def get_permissions(self):
+        permissions = []
+
+        for tag in ["uses-permission", "permission"]:
+            tag = self.xml.findall('.//' + tag)
```

Comment:
Thanks for the contribution @hatamiarash7 but can you not re-assigning `tag` everytime the loop runs ?